core/arch/arm/kernel/kern.ld.S

```diff
@@ -93,6 +93,9 @@ SECTIONS
 	.text : {
 		KEEP(*(.text._start))
 		__identity_map_init_start = .;
+		__text_data_start = .;
+		*(.text.data)
```

Comment:
If there's ever a function, static or global, named data it will go into this too.
How about using `.identify_map.data` instead? That way we'd have full control of what's going in here.

Reply:
Fair enough, and it also makes it obvious that the data need to be reachable using the identity mapping which is good.
Can I still keep `__text_data_start`/`__text_data_end`? To the "outside world" (tools etc.) it's essentially data embedded in `.text`.

Reply:
Sure